### DISCUSSION TITLE
my issue

### DISCUSSION THREAD

**Discussion by Jagadishkittu** [community/community#110407] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i have raised an issue regarding Dota2 game recalibration and i got banned 2 times for no reason
i dint got any solution to this or any updated msg
pls solve the problem , i dont need to hide my calibre , pls reset my calibre i struggled so much to get that calibre
![Screenshot 2024-02-28 210642](https://github.com/community/community/assets/161480882/2b952f6d-87ab-4c7f-b006-692983277638)

**Comment by HawkOne** [comment]
uhm... I'm sorry,
but what has that to do with Github and open source software.. ?

**Reply by Jagadishkittu** [reply]
Then how do i contact Dota2 service
i am unable to contact them
can u pls provide me any contact of the like email

**Comment by ettaboyle** [comment]
Hi @Jagadishkittu,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/zigbee2mqtt). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.


In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 


Best of luck!
